ci/jira_query.py

```diff
@@ -39,7 +40,10 @@
 FIELDS = 'summary,status,labels,customfield_10044,customfield_10045'
 
 # Labels in GitHub which this script will set/clear based on the logic below.
-MANAGED_LABELS = ('release-2.2', 'release-2.4', 'priority')
+MANAGED_LABELS_REGEX = re.compile(r'^(priority)|(release-(\d+\.\d+))$')
```

Comment:
I think the correct (simplified -- I don't see any use of capture groups for this one) RE here is:

```suggestion
MANAGED_LABELS_REGEX = re.compile(r'^(priority|release-\d+\.\d+)$')
```

I'd be interested though, why @ashleypittman didn't use an RE in the first place.  Maybe he's aware of something we are not.

Reply:
The capture groups don't serve any functional purpose.
I've added them as a matter of habit - it looks a bit more clear to me, but you are completely right that it doesn't really need to be here.

Reply:
@jzarzycki Please conversation resolving to the person that started the conversation once they have decided that the resolution meets their expectations.  Thanks.